cmd/gb/gb_test.go

```diff
@@ -451,7 +451,7 @@ func (t *T) mustBeEmpty(root string) {
 		found = append(found, path)
 		return nil
 	}
-	err = filepath.Walk(root, fn)
+	_ = filepath.Walk(root, fn)
```

Comment:
what about 

filepath.Walk(root, fn)

without assigning the unused value to anything?